packages/synapse-react-client/src/lib/containers/entity_finder/EntityFinder.tsx

```diff
@@ -208,43 +215,47 @@ export const EntityFinder: React.FunctionComponent<EntityFinderProps> = ({
 
   return (
     <SynapseErrorBoundary>
-      <div className="bootstrap-4-backport EntityFinder">
-        <div className="EntityFinder__Search" data-active={searchActive}>
+      <div className="EntityFinder">
+        <Box
+          display="flex"
+          justifyContent="flex-end"
+          alignItems="stretch"
+          pb={1}
+        >
           <>
             {searchActive ? (
               <Button
-                variant="gray-primary-500"
+                variant="outlined"
+                color="primary"
                 onClick={() => {
                   setSearchActive(false)
                   setSearchInput('')
                   setSearchTerms(undefined)
                 }}
+                startIcon={<ArrowBackOutlinedIcon />}
+                sx={{ flexShrink: 0, mr: 1 }}
               >
-                <Arrow arrowDirection="left" style={{ height: '16px' }} />
                 Back to Browse
               </Button>
             ) : (
               <Button
-                variant="sds-primary"
+                variant="contained"
+                color="primary"
                 className="EntityFinder__Search__SearchButton"
                 onClick={() => {
                   setSearchActive(true)
                   searchInputRef.current!.focus()
                 }}
+                startIcon={<SearchIcon />}
+                sx={{ flexShrink: 0 }}
               >
-                <IconSvg icon="search" />
                 {searchButtonText}
               </Button>
             )}
-            <span className="SearchIcon">
-              <IconSvg icon="search" />
-            </span>
-            <FormControl
-              role="textbox"
-              ref={searchInputRef}
-              aria-hidden={!searchActive}
+            <TextField
               className="EntityFinder__Search__Input"
-              type="search"
+              data-active={searchActive}
```

Comment:
Apply `data-active` to TextField so the flex-grow properties can be applied for the transition